app/uk/gov/hmrc/agentclientrelationships/auth/AuthActions.scala

```diff
@@ -156,7 +190,7 @@ trait AuthActions extends AuthorisedFunctions {
     authorised(
       Enrolment("HMRC-AS-AGENT")
         and AuthProviders(GovernmentGateway))
-      .retrieve(authorisedEnrolments) { enrolments =>
+      .retrieve(allEnrolments) { enrolments =>
```

Comment:
if we can reuse the existing private `AuthorisedAsClient(...)` then this would revert to previous `authorisedEnrolments`

Reply:
ok